lighthouse-core/audits/seo/is-crawlable.js

```diff
@@ -122,10 +122,14 @@ class IsCrawlable extends Audit {
           const robotsTxt = robotsParser(robotsFileUrl.href, artifacts.RobotsTxt.content);
 
           if (!robotsTxt.isAllowed(mainResource.url)) {
+            const line = robotsTxt.getMatchingLineNumber(mainResource.url);
             blockingDirectives.push({
               source: {
-                type: /** @type {'url'} */ ('url'),
-                value: robotsFileUrl.href,
+                type: /** @type {'source-location'} */ ('source-location'),
```

Comment:
this falls back to URL in non-devtools, right?

Reply:
correct (see https://github.com/GoogleChrome/lighthouse/pull/9354 for images)